<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Media Player</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/gui-icon2.png</normaloff>images/gui-icon2.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QToolButton" name="previousButton">
      <property name="toolTip">
       <string>Previous track</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{
border: none ;
background: transparent ;
}

QToolButton:pressed{
background-color : rgb(203, 203, 203)
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/control-skip-180.png</normaloff>images/control-skip-180.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="11">
     <widget class="QToolButton" name="muteButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/speaker-volume.png</normaloff>images/speaker-volume.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QToolButton" name="nextButton">
      <property name="toolTip">
       <string>Next track</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{
border: none ;
background: transparent ;
}

QToolButton:pressed{
background-color : rgb(203, 203, 203)
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/control-skip.png</normaloff>images/control-skip.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="10">
     <widget class="QComboBox" name="rateBox"/>
    </item>
    <item row="2" column="7">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>27</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="3">
     <widget class="QToolButton" name="stopButton">
      <property name="toolTip">
       <string>Stop</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{
border: none ;
background: transparent ;
}

QToolButton:pressed{
background-color : rgb(203, 203, 203)
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/control-stop-square.png</normaloff>images/control-stop-square.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="13" colspan="2">
     <widget class="QToolButton" name="fullscreenButton">
      <property name="toolTip">
       <string>Fullscreen (Press Esc key to exit)</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{
border: none ;
background: transparent ;
}

QToolButton:pressed{
background-color : rgb(203, 203, 203)
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/full-screen.png</normaloff>images/full-screen.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QToolButton" name="shuffleButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/shuffle.png</normaloff>images/shuffle.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>25</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" colspan="15">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QVideoWidget" name="video" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="14">
     <widget class="QSlider" name="timeSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="12">
     <widget class="QSlider" name="volumeSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="9">
     <widget class="QToolButton" name="snapButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/screenshot.png</normaloff>images/screenshot.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QToolButton" name="playButton">
      <property name="toolTip">
       <string>Play or Pause</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{
border: none ;
background: transparent ;
}

QToolButton:pressed{
background-color : rgb(203, 203, 203)
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/control.png</normaloff>images/control.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>25</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="14">
     <widget class="QLabel" name="track_time">
      <property name="toolTip">
       <string>Current time | Total time</string>
      </property>
      <property name="text">
       <string>00:00 | 00:00</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QComboBox" name="playbackBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionOpen_Playlist"/>
    <addaction name="actionSave_Playlist"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionMy_playlist"/>
    <addaction name="actionColour_Options"/>
   </widget>
   <widget class="QMenu" name="menuDownload">
    <property name="title">
     <string>Download</string>
    </property>
    <addaction name="actionDownload_Using_URL"/>
   </widget>
   <widget class="QMenu" name="menuBrowse">
    <property name="title">
     <string>Browse</string>
    </property>
    <addaction name="actionOpen_Browser"/>
   </widget>
   <widget class="QMenu" name="menuNotes">
    <property name="title">
     <string>Notes</string>
    </property>
    <addaction name="actionOpen_Notemaker"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuDownload"/>
   <addaction name="menuBrowse"/>
   <addaction name="menuNotes"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>My playlist</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListView" name="playlistView">
       <property name="toolTip">
        <string>Playlist</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background : trasperent
}
</string>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionOpen_Playlist">
   <property name="text">
    <string>Open Playlist</string>
   </property>
  </action>
  <action name="actionMy_playlist">
   <property name="text">
    <string>My_playlist</string>
   </property>
  </action>
  <action name="actionColour_Options">
   <property name="text">
    <string>Colour Options...</string>
   </property>
  </action>
  <action name="actionSave_Playlist">
   <property name="text">
    <string>Save Playlist</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDownload_Using_URL">
   <property name="text">
    <string>Download Using URL..</string>
   </property>
  </action>
  <action name="actionOpen_Browser">
   <property name="text">
    <string>Open Browser</string>
   </property>
  </action>
  <action name="actionOpen_Notemaker">
   <property name="text">
    <string>Open Notemaker</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
